frontend/components/organism/games/Spelling-bee/SpellingBee.jsx

```diff
@@ -0,0 +1,61 @@
+'use client';
```

Comment:
Why did you remove the tsx file?
Couldn't you just modify it to add what you needed?